src/modeler/AutoRest.Swagger/Validation/OperationIdSingleUnderscore.cs

```diff
@@ -39,7 +39,7 @@ public override bool IsValid(string entity)
         /// <summary>
         /// The severity of this message (ie, debug/info/warning/error/fatal, etc)
         /// </summary>
-        public override Category Severity => Category.Error;
+        public override Category Severity => Category.Warning;
```

Comment:
this may be a rule we need to enforce, due to AutoRest behavior, therefore leaving it as an error. What would happen at code generation time if there's more than 1 underscore?